actors/builtin/multisig/multisig_test.go

```diff
@@ -75,6 +76,111 @@ func TestConstruction(t *testing.T) {
 	})
 }
 
+func TestVesting(t *testing.T) {
+	actor := msActorHarness{multisig.MultiSigActor{}, t}
+
+	receiver := newIDAddr(t, 100)
+	anne := newIDAddr(t, 101)
+	bob := newIDAddr(t, 102)
+	charlie := newIDAddr(t, 103)
+	darlene := newIDAddr(t, 103)
+
+	const unlockDuration = 10
+	var multisigInitialBalance = abi.NewTokenAmount(100)
+	var nilParams = abi.MethodParams{}
+
+	builder := mock.NewBuilder(context.Background(), receiver).
+		WithCaller(builtin.InitActorAddr, builtin.InitActorCodeID).
+		WithEpoch(0).
+		// balance 0: current balance of the actor. receive: 100 the amount the multisig actor will be initalized with -- InitialBalance
+		WithBalance(abi.NewTokenAmount(0), multisigInitialBalance)
```

Comment:
This isn't how it will happen. The balance will be >= received, so for construction they will match.